test/ui-test/testSuites/suite_status/tst_communityFlows/test.feature

```diff
@@ -41,3 +41,14 @@ Feature: Status Desktop community
         | community_channel_name    | community_channel_description   |
         | test-channel    | Community channel description tested 1      |
 
+    Scenario Outline: Admin edits a community channel
+    	When the user creates a community named myCommunity, with description My community description, intro Community Intro and outro Community Outro
+		Then the user lands on the community named myCommunity
+		When the admin creates a community channel named test-channel, with description My description
+        Then the user lands on the community channel named test-channel
+        When the admin edits a community channel named <community_channel_name> to the name <new_community_channel_name>
+        Then the user lands on the community channel named <new_community_channel_name>
```

Comment:
nit: indent doesn't seem correct

Reply:
I think it should be fine now. I fixed my settings so that tabs do spaces instead.